src/components/UserInfo.js

```diff
@@ -1,19 +1,22 @@
 export default class UserInfo {
-  constructor({ nameSelector, jobSelector }) {
+  constructor({ nameSelector, jobSelector, avatarSelector }) {
     this._nameElement = document.querySelector(nameSelector);
     this._descriptionElement = document.querySelector(jobSelector);
+    this._avatarElement = document.querySelector(avatarSelector);
   }
 
   getUserInfo() {
     this._userData = {
       name: this._nameElement.textContent,
       description: this._descriptionElement.textContent,
+      avatar: this._avatarElement.src,
     };
     return this._userData;
   }
 
-  setUserInfo({ name, description }) {
+  setUserInfo({ name, about, avatar }) {
     this._nameElement.textContent = name;
-    this._descriptionElement.textContent = description;
+    this._descriptionElement.textContent = about;
+    this._avatarElement.src = avatar;
```

Comment:
Needs correction: It's needed to create a separate setter for avatar `setAvatar` and use it when needed to update the avatar, because user info and avatar are controlled by different forms.

When initializing user info you can use bit methods:

```js
userInfo.setUserInfo({ title, description })
userInfo.setAvatar(avatar)
```